.circleci/config/jobs/pre-flight-checks.yml

```
@@ -24,6 +24,13 @@ steps:
         . $BASH_ENV
         which circleci
         circleci version
-  - run: make ci-verify
+  - run:
+      name: Verify CircleCI
+      command: |
+        export CCI_PATH=/tmp/circleci-cli/$CIRCLECI_CLI_VERSION
+        echo "export PATH=$CCI_PATH:\$PATH" >> $BASH_ENV
+        set -x
+        . $BASH_ENV
```

nit: any reason not to simply do this? Perhaps there are some downstream jobs that also need the `circleci` command that I'm not aware of

```suggestion
        export CCI_PATH=/tmp/circleci-cli/$CIRCLECI_CLI_VERSION
        export PATH=$CCI_PATH:$PATH
        set -x
```